modules/etcd-defrag.adoc

```diff
@@ -6,10 +6,43 @@
 [id="etcd-defrag_{context}"]
 = Defragmenting etcd data
 
-Manual defragmentation must be performed periodically to reclaim disk space after etcd history compaction and other events cause disk fragmentation.
+Defragmentation of etcd data reclaims disk space after etcd history compaction and other events  cause disk fragmentation.
```

Comment:
```suggestion
Defragmentation of etcd data reclaims disk space after etcd history compaction and other events cause disk fragmentation.
```

Alt: Defragment etcd data to reclaim disk space after events that cause disk fragmentation, such as etcd history compaction.